server_windows_test.go

```diff
@@ -0,0 +1,87 @@
+//go:build windows
+// +build windows
```

Comment:
> If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns:
```
*_GOOS
*_GOARCH
*_GOOS_GOARCH
```
> where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).

So, these build tags should also be redundant.

Reply:
Ah, thanks for that. Wasn't aware it strips `_test`. 👍🏻